tests/invlet_test.cpp

```diff
@@ -724,25 +725,25 @@ static void merge_invlet_test( player &dummy, inventory_location from )
 
 #define invlet_test_autoletter_off( name, dummy, from, to ) \
     SECTION( std::string( name ) + " (auto letter off)" ) { \
-        get_options().get_option( "AUTO_INV_ASSIGN" ).setValue( "disabled" ); \
+        override_option _( "AUTO_INV_ASSIGN", "disabled" ); \
```

Comment:
It took me a second to parse what was happening here due to the fact that we use `_( )` for translation. Maybe calling the variable `_handle` or something would make what's happening a little more obvious?

Reply:
Right, that slipped my mind.  I renamed them all `opt`.